src/components/DocumentationTopic/Summary/Availability.vue

```diff
@@ -99,92 +97,77 @@ export default {
 
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
+$icon-size-default: 20px;
+$availability-info-spacing: 10px;
 
-.availability {
-  display: flex;
-  flex-flow: row wrap;
-  gap: 10px;
-  margin-top: 25px;
-}
-
-.badge {
-  margin: 0;
-}
-
-.technology {
+.technology, .platform {
   display: inline-flex;
   align-items: center;
 }
 
 .tech-icon {
   height: 12px;
   padding-right: 5px;
-  fill: var(--badge-color);
-
-  @include prefers-dark {
-    fill: var(--badge-dark-color);
-  }
-}
-
-.beta {
-  color: var(--color-badge-beta);
-
-  @include prefers-dark {
-    color: var(--color-badge-dark-beta);
-  }
-}
-
-.deprecated {
-  color: var(--color-badge-deprecated);
-
-  @include prefers-dark {
-    color: var(--color-badge-dark-deprecated);
-  }
+  --color-svg-icon: var(--color-figure-gray);
 }
 
 .changed {
   $-coin-spacer: 5px;
-  padding-left: $icon-size-default + ($-coin-spacer * 2);
+  padding-left: $icon-size-default - $-coin-spacer + 2;
+  border: none;
 
   &::after {
-    content: none;
+    // unset the global .changed style
+    all: unset;
   }
 
   &::before {
     @include coin($modified-svg, $icon-size-default);
-    left: $-coin-spacer;
+    margin: 0;
+    left: -$-coin-spacer;
 
     @include prefers-dark {
       background-image: $modified-dark-svg;
     }
   }
 
-  &-added {
-    border-color: var(--color-changes-added);
+  &-added::before {
+    background-image: $added-svg;
 
-    &::before {
-      background-image: $added-svg;
-
-      @include prefers-dark {
-        background-image: $added-dark-svg;
-      }
+    @include prefers-dark {
+      background-image: $added-dark-svg;
     }
   }
 
-  &-deprecated {
-    border-color: var(--color-changes-deprecated);
-
-    &::before {
-      background-image: $deprecated-svg;
+  &-deprecated::before {
+    background-image: $deprecated-svg;
 
-      @include prefers-dark {
-        background-image: $deprecated-dark-svg;
-      }
+    @include prefers-dark {
+      background-image: $deprecated-dark-svg;
     }
   }
+}
 
-  &-modified {
-    border-color: var(--color-changes-modified);
+.availability {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $availability-info-spacing;
+  margin-top: rem(15px);
+
+  & > * {
+    @include font-styles(body-reduced);
+    &::after {
+      content: '';
+      display: inline-block;
+      width: 1px;
+      height: 14px;
```

Comment:
Is this meant to match the font size or line height? You may want to consider using em units if that's the case.